views/js/qtiCommonRenderer/tpl/container.tpl

```diff
@@ -1,5 +1,9 @@
 {{~#equal contentModel "blockStatic"~}}
 <div class="qti-block">{{{body}}}</div>
+<svg class="qti-block-overlay">
+    <line x1="0" y1="100%" x2="100%" y2="0" style="stroke:rgb(0,0,0);stroke-width:1"/>
+    <line x1="0" y1="0" x2="100%" y2="100%" style="stroke:rgb(0,0,0);stroke-width:1"/>
+</svg>
```

Comment:
if this is specific to answer eliminating functionality, I would not put it in the common container.tpl otherwise all blockstatic QTI element will have this div as sibling although they are not using it. So I would recommend moving this directly to the answer elimitor functionality in the choice interaction itself. 
In this process, you could probably rename the qti-block-overlay to something more specific to answer eliminating if this overlay is only supposed to be used for answer elimination.